fediseer/messaging.py

```diff
@@ -88,7 +88,7 @@ def send_mastodon_pm(self, message, username, domain):
         document["object"]["tag"] = [
 			{
 			  "type": "Mention",
-			  "to": f"@{username}",
+			  "name": f"@{username}",
```

Comment:
see [Activity Vocabulary: 5.6 Mentions, Tags and Other Common Social Microsyntaxes](https://www.w3.org/TR/activitystreams-vocabulary/#microsyntaxes)